pkg/engine/task/task_pipe.go

```diff
@@ -104,7 +104,10 @@ func (pt PipeTask) Run(ctx Context) (bool, error) {
 	// 8. - Copy out the pipe files -
 	log.Printf("PipeTask: %s/%s copying pipe files", ctx.Meta.InstanceNamespace, ctx.Meta.InstanceName)
 	fs := afero.NewMemMapFs()
-	pipePod := podObj[0].(*corev1.Pod)
+	pipePod, ok := podObj[0].(*corev1.Pod)
```

Comment:
This should never happen since we create that same `corev1.Pod` a few lines above it